packages/markdown/__tests__/index.test.js

```diff
@@ -175,3 +176,94 @@ test('should strip dangerous img attributes', () => {
     '<img src="x"/>',
   );
 });
+
+describe('export multiple Markdown renderers', () => {
+  const { dash, hub, ast, md, html } = markdown.render;
```

Comment:
Do we *have* to export so many of these? Isn't it just `editor` and `html`? When given a string of markdown? Or are these extra ones mainly for internal/debugging usage?

Reply:
No, we don't. I have found a number of them to be quite useful, though a bunch of 'em are _definitely_ exploratory/placeholders. (This is, I'd imagine, where PDF conversion would happen, for example!) But yeah, this file definitely could definitely use some going over and simplification!